.github/workflows/gh-pages.yml

```diff
@@ -7,21 +7,49 @@ on:
 
   workflow_dispatch:
 
-jobs:
-  deployment:
-    runs-on: ubuntu-latest
+env:
+  DEPLOYMENT_DOMAIN: mocha.chroma-gaming.xyz
```

Comment:
i think we can move this env to github action env

Reply:
![image](https://user-images.githubusercontent.com/47074517/186130130-680923e6-c4b6-439b-82ef-00fd27b73897.png)
mmm which one? because i'm confused by so many env terminology in github 😢 
so far, i'm using `Secrets -> Actions` because that's what i got after searching at google😅

Reply:
use the secret > action